src/LlamaPayV2Factory.sol

```diff
@@ -25,6 +25,7 @@ contract LlamaPayV2Factory {
         unchecked {
             amtOfPayers++;
         }
+        param = address(0);
```

Comment:
?

Reply:
turns out it uses more gas after multiple calls. reverting